src/main/scala/gemmini/ExecuteController.scala

```diff
@@ -6,6 +6,7 @@ import chisel3.util._
 import GemminiISA._
 import Util._
 import freechips.rocketchip.config.Parameters
+import org.w3c.dom.css.Counter
```

Comment:
I'm guessing this was an accidental import? :)

Reply:
Thanks for the catch! This is from my editor and I will fix it in this version.